docs/ProductSummaryShelf.md

```diff
@@ -47,6 +47,8 @@ The Product Summary Shelf is the main block exported by the [Product Summary app
 | - | - | - | - |
 | `priceBehavior` |  `enum` | Determines whether the component should fetch the most up-to-date price (`async`) or (`default`). Remember to configure the [Search Result](https://vtex.io/docs/components/content-blocks/vtex.search-result@3.79.1/#configuration)'s `simulationBehavior` prop to `skip` and use the Product Price [`product-price-suspense`](https://github.com/vtex-apps/product-price/blob/master/docs/README.md) block to render a loading spinner while the price information is being fetched. | `default` |
 | `trackListName` | `boolean` | Determines whether the component should send the list name to the product page when the product summary is clicked. Disabling it will prevent the `productDetail` GTM event sent on the PDP to identify from which list the user navigated. | `true` |
+| `sponsoredBadgeLabel` | `String` | The text of the "Sponsored" tag, if applicable. | `"store/sponsored-badge.label"`| 
+| `sponsoredBadgePosition` | `enum` | The text of the "Sponsored" tag, if applicable. Possible values are `titleTop`, `containerTopLeft` and `none`. | `"titleTop"`| 
```

Comment:
you are using the same description for both

Reply:
Ooops, I'll change that! Thanks!!

Reply:
Done https://github.com/vtex-apps/product-summary/pull/390/commits/6d3e179ecf5e47ed5d57506fed0559228f6cb47e